h/static/styles/annotation-share-dialog.scss

```diff
@@ -0,0 +1,130 @@
+.share-dialog-wrapper.open .annotation-share-dialog {
+  display: flex;
+  flex-direction: column;
+}
+
+.annotation-share-dialog {
+  display: none;
+  position: absolute;
+  right: 0;
+  bottom: 130%;
+  z-index: 1;
+  background: $white;
+  border: 1px solid $gray-lighter;
+  border-radius: 2px;
+  width: 200px;
+  font-size: $body1-font-size;
+  cursor: default;
+  box-shadow: 0px 0px 4px 0px rgba(0,0,0,0.15);
+
+  &:after, &:before {
```

Comment:
I suggest adding a note here to explain what the `:after` and `:before` pseudo-selectors are for.